assets/components/featuredDigitalPack/featuredDigitalPack.jsx

```diff
@@ -19,6 +23,7 @@ type PropTypes = {
   headingSize: HeadingSize,
   countryGroupId: CountryGroupId,
   url: string,
+  abTest: ComponentAbTest | void,
```

Comment:
wouldn't `null` be better here?

Reply:
Agree. I was using `undefined` to make the field disappear in the component event in `sendTrackingEventsOnClick`, but I think that's leaking an implementation detail of that function. We've standardised on `null` across the codebase so I'll use that.

Reply:
Done.